source/sass/global/global.scss

```
@@ -0,0 +1,26 @@
+a {
+  text-decoration: none;
+}
+
+img {
+  max-width: 100%;
+  vertical-align: middle;
+}
+
+ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.visually-hidden {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  margin: -1px;
+  padding: 0;
+  border: 0;
+  white-space: nowrap;
+  overflow: hidden;
+  clip: rect(0 0 0 0);
+}
```

От этой глобальщины тоже можно избавится, сделав доступное скрытие не классом-хелпером, а элементом блока page. К тому же пора уже переходить с этого длинного анахронизма на современный вариант доступного скрытия:
```suggestion
.page__a11y-hidden {
  position: fixed;
  transform: scale(0);
}
```

---

Хм, кто бы знал что такой есть :) Спасибо за подсказку. А почему название класса такое странное?

---

если сделать доступное скрытие через элемент блока ```page``` это разве не увеличит зависимость?

---

Странное сокращение? A11y — accessibility. Это нумерочегототам (не помню название сокращений таких): вырезается всё слово, кроме первой и последней буквы, вместо вырезанного оставляют количество вырезанных букв. i18n — internationalization. l10n — localization. Это самые распространённые, но их куча в употреблении.


Зависимость от чего? Это же получается элемент главного блока страницы — page. По сути это дело страницы, что показывать, а что нет. По хорошему даже скрываемый доступно контент надо стилизовать так, будто он не скрывается. И лишь потом пройтись и помиксовать где надо этот бэм-элемент.